python/generate/templates/api_client.mustache

```diff
@@ -0,0 +1,725 @@
+# coding: utf-8
+{{>partial_header}}
+from __future__ import absolute_import
+
+import atexit
+import datetime
+import json
+from dateutil.parser import parse
```

Comment:
The order of these two imports is unnecessarily switched from the default template